src/java/org/apache/cassandra/cql3/restrictions/RestrictionSet.java

```diff
@@ -104,6 +104,17 @@ public List<ColumnMetadata> getColumnDefs()
     {
         return new ArrayList<>(restrictions.keySet());
     }
+    
+    @Override
+    public Collection<ColumnMetadata> getColumnDefinitions()
+    {
+        return restrictions.keySet();
+    }
+
+    public Map<ColumnMetadata, SingleRestriction> getRestrictions()
```

Comment:
We're already iterable do we need this?

Reply:
It actually looks like this is simply unused. Removing it...